jenkins-pipelines/performance/branch-perf-v15/scylla-enterprise/perf-regression/scylla-enterprise-perf-regression-latency-650gb-with-nemesis-rbno-disabled.jenkinsfile

```diff
@@ -0,0 +1,11 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+perfRegressionParallelPipeline(
+    backend: "aws",
+    test_name: "performance_regression_test.PerformanceRegressionTest",
+    test_config: """["test-cases/performance/perf-regression-latency-650gb-with-nemesis.yaml", "configurations/tablets_disabled.yaml", "configurations/disable_kms.yaml","configurations/disable_rbno.yaml"]""",
```

Comment:
tablets disabled - got me rethinking of it.
It was mostly relevant for us only for the runs of 2024.1, possibly for 2024.2.
However, longer term we don't want to run with tablets disabled.

However, with tablets I think this test is not needed (it doesn't use RBNO AFAIK, does it?).

Reply:
As I found RBNO is used when tablets enabled

Reply:
in the `2022.2` and `2023.1` versions RBNO is enabled by default for **replace** operation only

Reply:
@roydahan this question should go to tablet team. 

@bhalevy @asias, is disabling RBNO + enabling tablets makes any sense testing (and specifically operation performance test) ?

(the only thing I  can think of is, some keyspace are tablets some not, and one want to use streaming)

Reply:
rbno vs. streaming applies only to vnode keyspaces.
Handling tablets as part of or in response to node operations uses tablet migrations, nit rbno, nor legacy streaming node operations.

Reply:
o.k. so we have this job running only on vnodes